File: Wizard Battle Web API/DTOs/Transaction/DirectTransactionResponse.cs
namespace Wizard_Battle_Web_API.DTOs.Transaction
{
	public class DirectTransactionResponse
	{
		public int TransactionID { get; set; } = 0;

		public StaticPlayerResponse Player { get; set; } = null!;

		public StaticSkinItemResponse SkinItemID { get; set; } = null!;

		public int TotalCost = 0;

		public DateTime Created_At { get; set; }
	}
}

File: Wizard Battle Web Client/src/app/_models/Transaction/DirectTransactionResponse.ts
import { StaticPlayerResponse } from "../Player";
import { StaticSkinItemResponse } from "../SkinItem";

export interface DirectTransactionResponse {
    transactionID: number;
    player: StaticPlayerResponse;
    skinItem: StaticSkinItemResponse;
    totalCost: number;
    created_At: string;
}
